api/controllers/organization.js

```diff
@@ -0,0 +1,17 @@
+const organizationSerializer = require('../serializers/organization');
+const { User, Organization } = require('../models');
+const { wrapHandlers } = require('../utils');
+const { fetchModelById } = require('../utils/queryDatabase');
+
+module.exports = wrapHandlers({
+  async findAllForUser(req, res) {
+    const user = await fetchModelById(req.user.id, User, { include: [Organization] });
```

Comment:
Not a big deal and I think this is lifted from other places, but semantically, we want to be querying for `Organization`s that belong to a particular user, similar to how that `forUser` scope is implemented on a `Site`:

```
  Site.addScope('forUser', user => ({
    include: [{
      model: User,
      required: true,
      where: {
        id: user.id,
      },
    }],
  }));
```

Reply:
Scoped